src/Components/Aspire.Orleans.Server/AspireOrleansServerExtensions.cs

```diff
@@ -0,0 +1,214 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net;
+using Aspire.Orleans.Shared;
+using Azure.Data.Tables;
+using Azure.Storage.Blobs;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Orleans.Clustering.AzureStorage;
+using Orleans.Configuration;
+using Orleans.Reminders.AzureStorage;
+using static Aspire.Orleans.Shared.OrleansServerSettingConstants;
+
+namespace Aspire.Orleans.Server;
+
+public static class AspireOrleansServerExtensions
+{
+    private const string OrleansConfigKeyPrefix = "Orleans";
+
+    public static IHostApplicationBuilder UseAspireOrleansServer(this IHostApplicationBuilder builder)
```

Comment:
This should be Add, not Use.

Reply:
For server, to me, it doesn't make sense to use `Add` instead of `Use` since I don't expect anyone to run multiple silos in different cluster in the same program?

Reply:
@halter73 do you have thoughts here on `Add` vs `Use`?

Reply:
Considering this is on `IHostApplicationBuilder` and not `IServiceCollection`, I like `Use`. I see it as more similar to `IHostBuilder` than `IServiceCollection`, and we use `Use` as a prefix for a number of `IHostBuilder` extension methods. I don't think `IHostBuilder` has any extension methods that start with `Add`, but please remind me if I'm forgetting something.

The points about it being unlikely to have multiple calls to this in the same program and not needing to call another method later to enable it like you would for middleware are also pretty compelling to me. Why do you think it should start with `Add` @davidfowl?